analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -210,5 +211,16 @@ class SpdxDocumentFile(
     private fun getConcludedLicense(pkg: SpdxPackage): SpdxExpression? =
         pkg.licenseConcluded.takeIf { SpdxConstants.isPresent(it) }?.toSpdx()
 
-
+    /**
+     * Return a [RemoteArtifact] created with downloadLocation of the given package [SpdxPackage]
+     * if it's a file location or return en Empty RemoteArtifact.
+     * TODO: must be completed for other cases : use "sourceInfo" or "externalRefs"?
+     */
+    private fun buildSourceArtifact(pkg: SpdxPackage): RemoteArtifact {
+        return if (pkg.downloadLocation.startsWith("file:/")) {
```

Comment:
Per SPDX spec 2.1.1, [Package Download Location](https://github.com/spdx/spdx-spec/blob/v2.1.1/chapters/3-package-information.md) can be a uniform resource locator but the docs don't specify accepted formats - I am not sure `file:/` can be used for this field. Need to discuss this with my fellow SPDX maintainers in our weekly tech call.

Reply:
Personally, I'd be fine with committing this now, and eventually fixing it up later.